addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/job-ascan.html

```diff
@@ -26,6 +27,14 @@ <H1>Automation Framework - activeScan Job</H1>
       injectPluginIdInHeader:          # Bool: If set then the relevant rule Id will be injected into the X-ZAP-Scan-ID header of each request, default: false
       scanHeadersAllRequests:          # Bool: If set then the headers of requests that do not include any parameters will be scanned, default: false
       threadPerHost:                   # Int: The max number of threads per host, default: 2
+    policyDefinition:                  # The policy definition - only used if the 'policy' is not set
+      defaultStrength:                 # String: The default Attack Strength for all rules, one of Low, Medium, High, Insane (not recommended), default: Medium
+      defaultThreshold:                # String: The default Alert Threshold for all rules, one of Off, Low, Medium, High, default: Medium
+      rules:                           # A list of one or more active scan rules and associated settings which override the defaults
+      - id:                            # Int: The rule id as per https://www.zaproxy.org/docs/alerts/
+        name:                          # String: The name of the rule for documentation purposes - this is not required or actually used
```

Comment:
Can we make it more obvious that this is optional/doc/unused?

Something like "(Optional) String:"?

Reply:
Changed to "Comment: " - this will be generated when I implement the option for generating the yaml file from an existing configuration :)